### PR TITLE
modify container swagger files to include port and scheme fields

### PR DESCRIPTION
**Pull request by farazmsiddiqi**
update converters to handle scheme and port fields

# Description

made changes to the radius datamodel to include an optional port and scheme field that help enable DNS-SD. 

## Type of change

- This pull request is a minor refactor, code cleanup, test improvement, or other maintenance task and doesn't change the functionality of Radius (issue link optional).

Helps fix: radius-project/radius#6375 

### Summary
🚢🔧📝

This pull request adds the ability to specify custom ports and schemes for container endpoints in the Applications.Core resource provider. It updates the external and internal API models, the Swagger specification, and the JSON serialization and deserialization functions for the `ContainerPort` struct. It also fixes a JSON syntax error in the Swagger example file.

> _Sing, O Muse, of the skillful engineers who devised_
> _A wondrous feature for the `ContainerPort` struct,_
> _To let the users choose their ports and schemes with ease_
> _For their containerized applications, swift and robust._

### Walkthrough
*  Add `Port` and `Scheme` fields to the `ContainerPort` struct in the external API model, the internal data model, and the JSON serialization and deserialization functions ([link](https://github.com/project-radius/radius/pull/5887/files?diff=unified&w=0#diff-748054c59cb5c49f1136d2fa782e435ccfb3c80be34b3d784d1d7fde679653dbR73-R74), [link](https://github.com/project-radius/radius/pull/5887/files?diff=unified&w=0#diff-748054c59cb5c49f1136d2fa782e435ccfb3c80be34b3d784d1d7fde679653dbR187-R188), [link](https://github.com/project-radius/radius/pull/5887/files?diff=unified&w=0#diff-256a0f1ad5114a905914c6f931e4b42cc7f4c6cab143fcaebd4a77c48b2fdebeR329-R331), [link](https://github.com/project-radius/radius/pull/5887/files?diff=unified&w=0#diff-256a0f1ad5114a905914c6f931e4b42cc7f4c6cab143fcaebd4a77c48b2fdebeR337-R339), [link](https://github.com/project-radius/radius/pull/5887/files?diff=unified&w=0#diff-f404ae8ea064968eea9fa6e167913acc73be478dd3f36f7840767c9e7cd0d84dL533-R536), [link](https://github.com/project-radius/radius/pull/5887/files?diff=unified&w=0#diff-f404ae8ea064968eea9fa6e167913acc73be478dd3f36f7840767c9e7cd0d84dR552-R554), [link](https://github.com/project-radius/radius/pull/5887/files?diff=unified&w=0#diff-f404ae8ea064968eea9fa6e167913acc73be478dd3f36f7840767c9e7cd0d84dR561-R563), [link](https://github.com/project-radius/radius/pull/5887/files?diff=unified&w=0#diff-5275155acd7f7a59f0301cfc95f8dd38f72f22795e3197c6a150bae13dd5d5f3R100-R101))
*  Add `Port` and `Scheme` fields to the `ContainerPort` definition and example in the Swagger specification (`containers.json` and `ContainersListByScope.json`) ([link](https://github.com/project-radius/radius/pull/5887/files?diff=unified&w=0#diff-46f82ce5894ea72002dfdafceaa367b04650efac307bd067b150b296804e4549R388-R395), [link](https://github.com/project-radius/radius/pull/5887/files?diff=unified&w=0#diff-46f82ce5894ea72002dfdafceaa367b04650efac307bd067b150b296804e4549L779-R787), [link](https://github.com/project-radius/radius/pull/5887/files?diff=unified&w=0#diff-ff5028ba7f86daffa8d99a2707ba793f86360846fa9274c4dcdf1a0bfdf0f6c8L50-R52), [link](https://github.com/project-radius/radius/pull/5887/files?diff=unified&w=0#diff-ff5028ba7f86daffa8d99a2707ba793f86360846fa9274c4dcdf1a0bfdf0f6c8L102-R104))
*  Fix trailing commas in the Swagger specification (`containers.json` and `ContainersListByScope.json`) ([link](https://github.com/project-radius/radius/pull/5887/files?diff=unified&w=0#diff-46f82ce5894ea72002dfdafceaa367b04650efac307bd067b150b296804e4549L779-R787), [link](https://github.com/project-radius/radius/pull/5887/files?diff=unified&w=0#diff-ff5028ba7f86daffa8d99a2707ba793f86360846fa9274c4dcdf1a0bfdf0f6c8L102-R104))

### CODE REVIEW

**Review by nithyatsu — APPROVED** [review]

**Review by nithyatsu — CHANGES_REQUESTED** [review]
please make generate and push again.

**Review by nithyatsu — APPROVED** [review]
Approving, once @rynowak approves too, we can get this in.

**Review by rynowak — APPROVED** [review]

**Pull request merged**
